### DISCUSSION TITLE
[FULL REALESE] M.R.S. Titanic Nzp Map CODBO4

### DISCUSSION THREAD

**Discussion by Cybrock9000** [nzp-team/nzportable#810] [Map Releases]
### [DOWNLOAD](https://archive.org/details/m.-r.-s.-titanic-nzp-map-main)
You are the last person alive on the Titanic. You and your fellas (if you do multiplayer) will be fighting endless hoards of zombies and unraveling the secrets within. This map is a demake of the Voyage of Despair map on codbo4. This is also my first map and knowing me (which you dont) i go big on projects. I will make a copple updates here and there but this is (finally) 100% complete.
### pics ### 
![Screenshot 2024-06-25 231607](https://github.com/nzp-team/nzportable/assets/118949998/291dab77-f4dc-46d6-9bda-82b59dde3d50)
![Screenshot 2024-06-25 231545](https://github.com/nzp-team/nzportable/assets/118949998/8a33aa9d-c0ef-401b-a511-f78166dca237)
![Screenshot 2024-06-25 231356](https://github.com/nzp-team/nzportable/assets/118949998/84b4f400-4bee-45a9-9305-80bb1c2563cd)
![Screenshot 2024-06-25 231331](https://github.com/nzp-team/nzportable/assets/118949998/2664ee3b-38a5-4cb7-8c66-d53637dfd2ba)
![Screenshot 2024-06-25 231258](https://github.com/nzp-team/nzportable/assets/118949998/f2b42378-a0ff-48ef-8a5b-470295b21b38)
![Screenshot 2024-06-25 231240](https://github.com/nzp-team/nzportable/assets/118949998/ddab21ef-a0ac-4e44-8b05-a4557340d564)
![Screenshot 2024-06-25 231209](https://github.com/nzp-team/nzportable/assets/118949998/9282bb84-f0e6-4f65-aa2c-6cd3cf065177)
### other stuff
There is a ending to this map **SO DONT LOOK AT THESE INSTRUCTIONS IF YOU DONT WAND SPOILERS**
1. There are 10 teddy bears, shoot them
2. the door on the bow should be open (i have not tested this yet)

### READ THIS
The beginning is slow but once you get to round 5 it starts getting fast but try to get the power on (where the sentinel artifact was) and open up doors (waypoints are finicky on the 3ds)

### Works well and tested on:
PC
3DS

**Comment by Cybrock9000** [comment]
### Bugfix
Fixed the waypoints (for the most part)

**Comment by sealteamaqua** [comment]
i wanna dowload this, how?

**Reply by Cybrock9000** [reply]
https://github.com/nzp-team/nzportable/assets/118949998/efb7ac7d-d2cd-4d8b-b4ee-591fcf41ec0e

**Reply by brenan147** [reply]
can you upload the map to the website version? https://nzp.gay/

**Comment by Cybrock9000** [comment]
### QOL Update
Moved zombies spawners from the poop deck to the bow to make the beginning faster

**Comment by sealteamaqua** [comment]
good, was taking too long xD
could you brighten it a bit

**Reply by Cybrock9000** [reply]
I haven’t got to that yet, but I am going to

**Comment by Cybrock9000** [comment]
### Adjusted this page to meet the "Map Release Standardization" and QOL Update
brightened some dark arias and fixed some spawnpoints

**Comment by Cybrock9000** [comment]
### Fixed Zombie Spawning
i looked through some of the maps (ndu) and found out how the spawning should be, so now it is finished. (but i do want to make a decorated version for pc only)

**Reply by MisaelJama** [reply]
It looks too cool could you make it for psp please🥺

**Reply by MotoLegacy** [reply]
It should already work on PSP... did you test it?

**Reply by MisaelJama** [reply]
yes but it doesn't work

**Reply by MotoLegacy** [reply]
![image](https://github.com/user-attachments/assets/53ba06de-6247-4977-bbcb-aa5437108a4d)

Fixed with https://github.com/nzp-team/vril-engine/commit/8ec6018ba22907cdd64d1d714dde27b49d5e67e6 . Game will update in 3 and a half hours, download the new version when this happens.

**Reply by MisaelJama** [reply]
Thank you very much, thank you, I appreciate it because I like this game too much and I wanted to try this map.☺️

**Reply by MotoLegacy** [reply]
If there are any other custom maps with issues on PSP please open an [issue](https://github.com/nzp-team/nzportable/issues) and follow the template accordingly. I will fix them from the game itself when I can.

**Reply by MisaelJama** [reply]
I'll let you know

**Reply by MisaelJama** [reply]
What do I have to do to make it work because I already downloaded it and it still doesn't work?

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not on archive.org

**Reply by Cybrock9000** [reply]
fixed it

**Comment by tuffert155** [comment]
Cool as hell, I am however an idiot and have no idea whatsoever on how to install custom maps to nzp other then playing it via multiplayer.

**Reply by Jeyphr** [reply]
Have I got the video for you! : https://www.youtube.com/watch?v=FVf6Ww9R_rY

**Reply by tuffert155** [reply]
Oh thank you, I eventually figured it out though,

**Reply by Cybrock9000** [reply]
thanks for playing
